packages/openapi-v3/src/__tests__/unit/enhancers/spec-enhancer.extensions.unit.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {OASEnhancerService} from '../../..';
+import {SpecServiceApplication} from './fixtures/application';
+
+describe('spec-enhancer-extension-point', () => {
+  let app: SpecServiceApplication;
+  let specService: OASEnhancerService;
+
+  beforeEach(givenAppWithSpecComponent);
+  beforeEach(findSpecService);
+
+  it('setter - can set spec', async () => {
+    const EXPECTED_SPEC = {
+      openapi: '3.0.0',
+      info: {title: 'LoopBack Application', version: '1.0.0'},
+      // setter adds paths spec based on the default spec
+      paths: getSamplePathSpec(),
+    };
+    const PATHS_SPEC = getSamplePathSpec();
+    specService.spec = {...specService.spec, paths: PATHS_SPEC};
+    expect(specService.spec).to.eql(EXPECTED_SPEC);
+  });
+
+  it('generateSpec - loads and create spec for ALL registered extensions', async () => {
```

Comment:
loads and creates spec

OR

loads and applies spec  

Do we want to say create or apply? You have methods like ApplyEnhancer etc